programs/util.h

```diff
@@ -195,10 +195,7 @@ UTIL_STATIC void* UTIL_realloc(void* ptr, size_t size)
 /*-****************************************
 *  String functions
 ******************************************/
-/*
- * A modified version of realloc().
- * If UTIL_realloc() fails the original block is freed.
-*/
+/* supports a==NULL or b==NULL */
 UTIL_STATIC int UTIL_sameString(const char* a, const char* b)
 {
     assert(a!=NULL && b!=NULL);  /* unsupported scenario */
```

Comment:
Currently, if `a == NULL` *or* `b == NULL`, this line rises assert.
But above comment and the following `if()` clause seems to support these cases.

If so, this assert should refuse only `a == NULL` *and* `b == NULL` case.
Therefore, expected condition may be `assert(! (a == NULL && b == NULL));`.
In short, `assert(a != NULL || b != NULL);`.

Reply:
Good point @t-mat !
Thanks, let's fix that as you suggest.

On a related tangent, since this `assert()` was never triggered so far, it's questionable if this is a use case worth supporting, since it seems unused.

But anyway, this function was not the point of this PR, 
I just wanted to fix the code comment which was nonsense.